pkg/api/errors/messages.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+const (
+	// StateTransaction
+	MsgStateTransactionsNotSupported = "state store %s doesn't support transaction"
```

Comment:
This might make more sense to live in the `api/errors/state.go`? I doubt this will be reused by other building blocks, so I don't think it should live in it's own generic file.